client/js/widgets/widget.js

```diff
@@ -457,8 +457,14 @@ export class Widget extends StateManaged {
     if(!this.get('clickable') && !(mode == 'ignoreClickable' || mode =='ignoreAll'))
       return true;
 
-    if(this.get('clickSound'))
-      this.set('audio','source: ' + this.get('clickSound') + ', type: audio/mpeg , maxVolume: 1.0, length: null, player: null');
+    if(this.get('clickSound')) {
+      this.set('audio', JSON.stringify({
```

Comment:
Every `set` needs to be `await set`. Apparently it was also missing in the old code. :grin:

Reply:
I REALLY appreciate the feedback.  Sadly I need to leave right now, otherwise I would jump into this.